build-system/tasks/bundle-size.js

```diff
@@ -208,7 +213,7 @@ function getLocalBundleSize() {
       cyan(shortSha(gitCommitHash())) + '.'
     );
   }
-  getBrotliBundleSize();
+  getBrotliBundleSizes();
```

Comment:
What's the point of this call? Should this function be returning something? JSDoc would help, but also not clear if there's any point to this function call

Reply:
`getBrotliBundleSizes` prints its own logs. @rsimha added the `getLocalBundleSize()` method originally, I'm just updating here

Reply:
So the point of this is just to log stuff?

Reply:
Yes, it's a debugging thing

Reply:
:+1: